### PR TITLE
[RTail] Deprecate

### PR DESCRIPTION
**Pull request by nervo**

### CODE REVIEW

File: manala.skeleton/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Removed
+- Rtail support (deprecated)
```
Comment (chalasr) [inline manala.skeleton/CHANGELOG.md]: I'd say `changed` for the deprecation time, and add a `removed` when truly remove it (next major?)

Comment (nervo) [reply manala.skeleton/CHANGELOG.md]: Oh dear, i already had an oral 👍 ...

Comment (nervo) [reply manala.skeleton/CHANGELOG.md]: Anyway, you're absolutely right !

Comment (nervo) [reply manala.skeleton/CHANGELOG.md]: See: https://github.com/manala/ansible-roles/commit/769faf7df7525382f1d06d112ad18c14540adf51

**Pull request merged**